doc/Developing.md

```diff
@@ -19,6 +42,27 @@ So far it has been confirmed that there are some problems and errors using nodej
 
 [nvm](https://github.com/creationix/nvm) is recommended to easily switch between node versions.
 
+## Quick Start
+
+To build and run the browser example:
+
+    git clone https://github.com/theia-ide/theia \
+    && cd theia \
+    && yarn \
+    && cd examples/browser \
```

Comment:
Question: shouldn't we add `yarn run rebuild:browser` here? Just to be able to switch between electron and the browser versions.

Reply:
I meant one above, sorry.

Reply:
One does not need to run it to start the browser example after cloning. If one wants to run not from the scratch there is a section covering it: https://github.com/pierregoutagny/theia/blob/improve-developing-md/doc/Developing.md#run-the-browser-based-example-application

Reply:
True. Does the `rebuild:electron` mandatory when building the electron example from a fresh clone? I am just asking because it seems to be inconsistent somehow.

Reply:
yes, for electron is mandatory